setup.cfg

```diff
@@ -34,6 +34,7 @@ install_requires =
     aiohttp
     asgiref
     airflow-provider-fivetran
+    openlineage-airflow>=0.18.0
```

Comment:
Does this mean users cant use async operator without installing `openlineage-airflow` package?

Reply:
+1, it should be an optional dependency.